gui/window.go

```diff
@@ -800,3 +811,39 @@ func GetThankYouMessage() string {
 
 	return text
 }
+
+func displayErrorDialog(err error) {
+	contentBox, err := gtk.BoxNew(gtk.ORIENTATION_VERTICAL, 0)
+	contentBox.SetHAlign(gtk.ALIGN_FILL)
+	if err != nil {
+		log.Error("Error creating box", err)
+		return
+	}
+
+	label, err := gtk.LabelNew(log.GetCrashInfoMsg())
+	if err != nil {
+		log.Error("Error creating label", err)
+		return
+	}
+	label.SetHAlign(gtk.ALIGN_CENTER)
+	label.SetMarginBottom(common.TopBottomMargin)
+	label.SetSelectable(true)
+	contentBox.PackStart(label, true, true, 0)
+
+	title := utils.Locale.Get("Sorry, something went wrong...")
```

Comment:
Maybe we just say "Something went wrong"? Debating the "Sorry" portion.

Reply:
Done.